Platform/Apple/virtual/src/font/fontEngine.s

```diff
@@ -1573,9 +1544,7 @@ Wp_eChr1 CLC
 	JMP PlotFnt	;plot the character
 
 ;Change Font
-Wp_CFnt	TXA 		;restore alpha char
-	SEC
-	SBC #$30	;change Chr"#" to Val#
+Wp_CFnt	SBC #$30	;change Chr"#" to Val#
 	AND #$03	;mask off digit
 ;;pf: this CMP is broken
```

Comment:
True. In fact, it appears we don't support change-font at all (nor need we). Removed.